### PR TITLE
Updated French Translation

### PR DESCRIPTION
**Pull request by LordAmaury**
Many untranslated strings added, many existing translation changes to consider.

### CODE REVIEW

File: language-pack-fr/data/messages/breach.dat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	messages: [
```
Comment (joshuaskelly) [inline language-pack-fr/data/messages/breach.dat]: Please put quotes around JSON properties. Our JSON serializer can handle this, but it makes life easier and JSON linting possible.

E.g.
```json
{
    "messages": []
}
```

File: language-pack-fr/data/messages/camp-note1.dat [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-	"repeats": true,
-	"messages": [
-		[
-			"Il y a une rumeur qui dit que ce lieu guarde le meilleur artéfact du vieil empire des mages. Il est dit que cet objet est assez puissant pour modifier la réalité. -N"
-		]
-	]
-}
+{
+	repeats: true,
+	messages: [
```
Comment (joshuaskelly) [inline language-pack-fr/data/messages/camp-note1.dat]: Missing quotes.

**Review by joshuaskelly — CHANGES_REQUESTED** [review]
Can you make sure your JSON properties are quoted? Otherwise looking good! 👍

**Review by louiscarl — APPROVED** [review]
Very welcome changes, thanks so much!

**Comment by LordAmaury** [comment]
Files updated to include JSON quotes. Tell me if something is missing I checked all files one by one.

**Review by joshuaskelly — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**